frontend/public/module/k8s/types.ts

```diff
@@ -494,7 +494,7 @@ export type JobTemplate = {
     backoffLimit?: number;
     completions?: number;
     manualSelector?: boolean;
-    parallelism?: boolean;
+    parallelism?: number;
```

Comment:
this should be a number

> The requested parallelism (.spec.parallelism) can be set to any non-negative value. If it is unspecified, it defaults to 1. If it is specified as 0, then the Job is effectively paused until it is increased.

source: https://kubernetes.io/docs/concepts/workloads/controllers/job/#controlling-parallelism